bin/x_04b_all_vars

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+echo "This will take a while ... grab a coffee/tea/water"
+mkdir -p logs
+mkdir -p plots
+N_JOBS=4
+
+i=0
+for var in MET HT ST WPT MT; do
+	echo "Plotting diff. x-section for distribution: $var"
+	nohup time python src/cross_section_measurement/04_make_plots_matplotlib.py -v $var -c 13 -p data/normalisation/background_subtraction/ -o plots/background_subtraction &> logs/04_${var}_plot_13TeV.log & # -a  <--add this option for additional plots
+	let i+=1
+	if (( $i % N_JOBS == 0 ))
+	then
+		echo "Waiting on the above to finish."
+    	wait;
+	fi
+done
+
+echo "Visible phase space"
+for var in MET HT ST WPT; do
+	echo "Plotting diff. x-section for distribution: $var"
+	nohup time python src/cross_section_measurement/04_make_plots_matplotlib.py --visiblePS -v $var -c 13 -p data/normalisation/background_subtraction/ -o plots/background_subtraction &> logs/04_${var}_plot_13TeV_vis.log & # -a  <--add this option for additional plots
```

Comment:
The output directory when run with this script is different to that when 04 is run interactively.  Not sure if this is an issue...